train.lic

```diff
@@ -106,7 +106,8 @@ class CrossingTrain
   end
 
   def setup
-    @hometown = get_settings.hometown
+    @settings = get_settings
+    @hometown = @settings.hometown
```

Comment:
Does this still need to be an instance var?

Reply:
I removed @hometown = @settings.hometown. @settings is called in function setup so still need as instance var.